docs/user-guide/platform-cleanup.md

```diff
@@ -0,0 +1,179 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Platform Cleanup
+
+During the lifecycle of your projects, various resources are created within the KubeRocketCI platform. There may come a time when you need to clean up resources that are no longer necessary. This could be due to various reasons such as deciding to move applications out of KubeRocketCI management, or compliance with specific regulations or standards necessitating the move of the application to a different environment that meets those requirements. Besides, removing applications without cleanup might cause issues when creating a resource with the same name. All these cases could serve as a reason for platform cleanup.
```

Comment:
here you need to write that if you want to re-add a component with the same name, cleaning artifacts is mandatory for the correct operation of the platform